__tests__/collection.test.ts

```diff
@@ -435,3 +435,28 @@ describe('random thisArg tests', () => {
 		}, array);
 	});
 });
+
+describe('ensure() tests', () => {
+	const coll = new Collection();
+	coll.set('a', 1);
+	coll.set('b', 2);
+
+	test('set new value if key does not exist', () => {
+		coll.ensure('c', () => 3);
+		expect(coll.size).toStrictEqual(3);
+		expect(coll.has('c')).toBeTruthy();
+	});
+
+	test('return existing value if key exists', () => {
+		const ensureB = coll.ensure('b', () => 3);
+		const getB = coll.get('b');
+		expect(ensureB).toBe(getB);
```

Comment:
It should check with static values, otherwise in the case `ensure` changed the value, we wouldn't know.
```suggestion
		expect(ensureB).toStrictEqual(2);
		expect(getB).toStrictEqual(2);
```